application-collabora-api/src/main/java/com/xwiki/collabora/configuration/CollaboraConfiguration.java

```diff
@@ -52,4 +52,13 @@
      * @since 1.1
      */
     boolean isEnabled();
+
+    /**
+     * Get the defined Collabora server. Fallback on the main wiki server configuration in case it was not defined at
+     * the wiki level.
+     *
+     * @return the Collabora Online server defined in the configuration
+     * @since 1.1
+     */
+    String getServer();
```

Comment:
It's not clear at all what the return string is supposed to be. How do you "define" a Collabora server? If the returned string represents an URL then the method name and documentation should indicate this (e.g. ``getServerURL``). Even if the returned value is not an URL I feel the method name should still be a bit more explicit (e.g. ``getServerName``, ``getServerId``, etc.).